src/app/workbench/sidebar/CreateNextWorkbenchSidebar.tsx

```diff
@@ -126,7 +127,7 @@ export function CreateNextWorkbenchSidebar({ workbench }: ICreateNextWorkbenchSi
                     }, 0);
                   }}
                 >
-                  {availableViews
+                  {/* {availableViews
```

Comment:
delete if not needed?

Reply:
good point, but also made me realize we dont need this whole file anymore, so I deleted it!